agent/installer/registry.go

```diff
@@ -11,18 +11,26 @@ import (
 type osk8sInstaller interface{}
 type k8sInstallerMap map[string]osk8sInstaller
 type osk8sInstallerMap map[string]k8sInstallerMap
-type filterBundlePair struct {
+type filterOsBundlePair struct {
 	osFilter string
 	osBundle string
 }
-type filterBundleList []filterBundlePair
+
+type filterK8sBundle struct {
+	k8sFilter string
+}
```

Comment:
Question: Is there an advantage of defining a struct type to hold only a single string field?
could we instead have?
```
filterk8s string
```

Reply:
It was like that at the beginning but with Anusha we decided to put it into a struct in order to be consistent with the OS bundle struct.